src/utils/dom.js

```diff
@@ -253,3 +262,23 @@ export const getTabables = (el = document) =>
   selectAll(TABABLE_SELECTOR, el)
     .filter(isVisible)
     .filter(i => i.tabIndex > -1 && !i.disabled)
+
+// Attempt to focus an element, and return `true` if successful
+export const attemptFocus = (el, options = {}) => {
+  if (isElement(el) && el.focus) {
+    try {
+      el.focus(options)
+    } catch {}
+  }
```

Comment:
Actually the outer `if` isn't needed here, as the `try/catch` will handle failed attempts to focus non-focusable elements.